plugins/woocommerce/client/blocks/bin/gen-block-list-doc.js

```diff
@@ -168,7 +168,11 @@ ${ blockjson.description || '' }
 
 -	**Name:** ${ blockjson.name }
 -	**Category:** ${ blockjson.category || '' }
--   **Ancestor:** ${ blockjson.ancestor || '' }
+-   **Ancestor:** ${
+		( Array.isArray( blockjson.ancestor ) &&
+			blockjson.ancestor.join( ', ' ) ) ||
+		''
+	}
```

Comment:
The ancestor is always array according to [the schema](https://raw.githubusercontent.com/WordPress/gutenberg/trunk/schemas/json/block.json). Are we not using array for that property somewhere?

Reply:
It can also be undefined. Having said that, in af840a0c6e07d77f94c5a68ec877a54e9d811be0 I refactored the code to use an optional chaining operator so it's easier to read.